### ISSUE TITLE
Application failed to start in seedstack 20.7

### ISSUE THREAD

**Issue by vincentBENNER** [seedstack/swagger-addon#9]
Hello,

I've moved my application from seedstack 19.11 to seedstack 20.7.
It includes webservices and api, and i also use swagger.

When i try to start the application i've got the following error 

```
Caused by: java.lang.NoSuchMethodError: org.codehaus.stax2.ri.SingletonIterator.create(Ljava/lang/Object;)Lorg/codehaus/stax2/ri/SingletonIterator;
	at com.ctc.wstx.util.DataUtil.singletonIterator(DataUtil.java:41)
	at com.ctc.wstx.evt.CompactStartElement.getAttributes(CompactStartElement.java:100)
	at com.sun.xml.ws.policy.sourcemodel.XmlPolicyModelUnmarshaller.getAttributeByName(XmlPolicyModelUnmarshaller.java:229)
	at com.sun.xml.ws.policy.sourcemodel.XmlPolicyModelUnmarshaller.initializeNewModel(XmlPolicyModelUnmarshaller.java:114)
	at com.sun.xml.ws.policy.sourcemodel.XmlPolicyModelUnmarshaller.unmarshalModel(XmlPolicyModelUnmarshaller.java:80)
	at com.sun.xml.ws.policy.jaxws.SafePolicyReader.readPolicyElement(SafePolicyReader.java:254)
	at com.sun.xml.ws.policy.jaxws.PolicyWSDLParserExtension.definitionsElements(PolicyWSDLParserExtension.java:398)
	at com.sun.xml.ws.wsdl.parser.DelegatingParserExtension.definitionsElements(DelegatingParserExtension.java:65)
	at com.sun.xml.ws.wsdl.parser.FoolProofParserExtension.definitionsElements(FoolProofParserExtension.java:70)
	at com.sun.xml.ws.wsdl.parser.WSDLParserExtensionFacade.definitionsElements(WSDLParserExtensionFacade.java:108)
	at com.sun.xml.ws.wsdl.parser.RuntimeWSDLParser.parseWSDL(RuntimeWSDLParser.java:449)
	at com.sun.xml.ws.wsdl.parser.RuntimeWSDLParser.parse(RuntimeWSDLParser.java:314)
	at com.sun.xml.ws.server.EndpointFactory.getWSDLPort(EndpointFactory.java:802)
	at com.sun.xml.ws.server.EndpointFactory.create(EndpointFactory.java:241)
	at com.sun.xml.ws.server.EndpointFactory.createEndpoint(EndpointFactory.java:134)
	at com.sun.xml.ws.api.server.WSEndpoint.create(WSEndpoint.java:548)
	at com.sun.xml.ws.api.server.WSEndpoint.create(WSEndpoint.java:530)
	at org.seedstack.ws.internal.WSPlugin.createWSEndpoint(WSPlugin.java:220)
	at org.seedstack.ws.web.internal.WSWebPlugin.start(WSWebPlugin.java:112)
	at io.nuun.kernel.core.internal.KernelCore.bindAndStartPlugins(KernelCore.java:312)
	at io.nuun.kernel.core.internal.KernelCore.start(KernelCore.java:291)
	at org.seedstack.seed.core.internal.init.KernelManager.createKernel(KernelManager.java:68)
	at org.seedstack.seed.core.Seed.createKernel(Seed.java:291)
	at org.seedstack.seed.web.internal.SeedServletContainerInitializer.onStartup(SeedServletContainerInitializer.java:40)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5132)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
```

It seems to be due to a conflict with some libraries used by swagger (stax2-api 3.1.4 & woodstox-core 5.0.3) and other version of these librairies from web-services-addon 3.0.4 (with new metro version in seedstack 20.7)

**Comment by hervestern** [comment]
Excluding the stax2-api library from the the swagger-addon dependency solves the application startup error. 
Still, swagger access will fail due to incompatibilities between the jackson dependencies pulled by seed and swagger-jersey2-jaxrs. 
This can be solved by upgrading the addon.

**Comment by Nheverest** [comment]
Excluding the stax2-api library from the the swagger-addon dependency solves the application startup error.
Still, swagger access will fail due to incompatibilities between the jackson dependencies pulled by seed and swagger-jersey2-jaxrs.
This can be solved by upgrading the addon.

**Comment by adrienlauer** [comment]
Fixed by #10
